src/resources/resources/assets/waila/lang/zh_cn.json

```diff
@@ -1,389 +1,400 @@
+    "config.waila.plugin_waila.show_icon"                        : "显示图标",
+    "config.waila.plugin_waila.icon_position"                    : "图标位置",
+    "config.waila.plugin_waila.icon_position_top"                : "顶部",
+    "config.waila.plugin_waila.icon_position_middle"             : "中间",
+    "config.waila.plugin_waila.icon_position_bottom"             : "底部",
+  
+    "config.waila.plugin_harvest"                                : "收获",
+    "config.waila.plugin_harvest.enabled"                        : "显示收获物品",
+    "config.waila.plugin_harvest.display_mode"                   : "显示模式",
+    "config.waila.plugin_harvest.display_mode_modern"            : "现代",
+    "config.waila.plugin_harvest.display_mode_classic"           : "经典",
+    "config.waila.plugin_harvest.display_mode_classic_minimal"   : "经典简约",
+    "config.waila.plugin_harvest.creative"                       : "在创造模式显示？",
+  
+    "config.waila.plugin_wailax"                                 : "Waila附加",
+    "config.waila.plugin_wailax.energy"                          : "能量",
+    "config.waila.plugin_wailax.energy.enabled_block"            : "显示方块能量",
+    "config.waila.plugin_wailax.energy.enabled_entity"           : "显示实体能量",
+    "config.waila.plugin_wailax.energy.blacklist"                : "能量黑名单",
+    "config.waila.plugin_wailax.progress"                        : "制作进度",
+    "config.waila.plugin_wailax.progress.enabled_block"          : "显示方块合成进度",
+    "config.waila.plugin_wailax.progress.enabled_entity"         : "显示实体合成进度",
+    "config.waila.plugin_wailax.progress.blacklist"              : "合成进度黑名单",
+    "config.waila.plugin_wailax.item"                            : "物品",
+    "config.waila.plugin_wailax.item.enabled_block"              : "显示方块物品内容",
+    "config.waila.plugin_wailax.item.enabled_entity"             : "显示实体物品内容",
+    "config.waila.plugin_wailax.item.nbt"                        : "同步NBT数据",
+    "config.waila.plugin_wailax.item.display_mode"               : "显示模式",
+    "config.waila.plugin_wailax.item.display_mode_desc"          : "网格: 显示为网格，就像库存的工作原理一样\n列表：显示为带内联名称的垂直列表\n动态：超过最大高度时从列表切换到网格",
+    "config.waila.plugin_wailax.item.display_mode_grid"          : "网格",
+    "config.waila.plugin_wailax.item.display_mode_list"          : "列表",
+    "config.waila.plugin_wailax.item.display_mode_dynamic"       : "动态",
+    "config.waila.plugin_wailax.item.max_height"                 : "最大高度",
+    "config.waila.plugin_wailax.item.sort_by_count"              : "按计数排序",
+    "config.waila.plugin_wailax.item.blacklist"                  : "物品内容黑名单",
+    "config.waila.plugin_wailax.fluid"                           : "流体",
+    "config.waila.plugin_wailax.fluid.enabled_block"             : "显示方块流体内容",
+    "config.waila.plugin_wailax.fluid.enabled_entity"            : "显示实体流体内容",
+    "config.waila.plugin_wailax.fluid.display_unit"              : "显示装置",
+    "config.waila.plugin_wailax.fluid.display_unit_millibuckets" : "毫桶？ (mB)",
```

Comment:
https://feed-the-beast.fandom.com/wiki/Milibuckets_(mB) Hope it will be helpful for your translation
However, I haven’t found anyone who has translated this text so far, so I recommend not to translate it.

Reply:
Yes, I couldn't find any translations related to Milibuckets (mB). I think your suggestion is very useful